dicepy/__init__.py

```diff
@@ -15,6 +15,35 @@ class MathType(Enum):
     add = '+'
     subtract = '-'
 
+class Roll:
+    keep_type = None
+    keep_amount = 1
+    dice = None
+    value = None
+
+    def __init__(self, sides: int, dice_count: int, keep_type: KeepType = None, keep_amount: int = 1):
+        self.dice = [randint(1, sides) for _ in range(dice_count)]
+        self.keep_type = keep_type
+        if keep_type == None:
+            self.value = sum(self.dice)
+        # 3d6k2 and 3d6kh2
+        elif keep_type == KeepType.highest or keep_type == KeepType.keep:
+            self.keep_amount = 2
+            self.value = sum(sorted(self.dice)[-self.keep_amount:])
+        else: # KeepType.lowest
```

Comment:
Since `self.keep_amount` is only being set in the first `elif`, it's not being set here - meaning it's implicit 1. I pushed a test case up to my `main` branch which should replicate this issue. You can test with `python -munittest` in the root of the project once you merge against it.